frame/support/procedural/src/construct_runtime/expand/origin.rs

```diff
@@ -182,15 +182,15 @@ pub fn expand_outer_origin(
 		// For backwards compatibility and ease of accessing these functions.
 		#[allow(dead_code)]
 		impl Origin {
-			/// Create with system none origin and `frame-system::Config::BaseCallFilter`.
+			/// Create with system none origin and [`frame_system::Config::BaseCallFilter`].
```

Comment:
You need to use `#system_path`

Reply:
Oh... Can you elaborate more what you mean by system_path?

Reply:
@thiolliere already has written this for you: https://github.com/paritytech/substrate/issues/9962#issuecomment-949419511

Reply:
I actually tried out what @thiolliere suggested, it wasn't outputting the right docs and I ran into errors too

Reply:
Then you need to fix this.

Reply:
Oh... OK!
Will research on how to fix that.

Reply:
I have tried modifying what i did earlier @bkchr 
Can you take a look?

Reply:
it doesn't work for me, did you tested your PR manually before requesting review ?
![PR](https://user-images.githubusercontent.com/10416848/142324538-a4726dfa-6baa-43b7-b57b-a5057db316c3.png)

Reply:
my bad. @thiolliere will push a fix for this

Reply:
@thiolliere  i have fix this in my latest push. You can check now

Reply:
Still awaiting your reviews @thiolliere @bkchr

Reply:
we will review when we get time